app/src/components/Accordion/theme.ts

```diff
@@ -15,7 +15,7 @@ export const accordionTheme: keepAccordionTheme = {
   icon: {
     base: 'transition-transform duration-300',
     rotated: {
-      full: 'rotate-180',
+      full: 'rotate-225',
```

Comment:
Tailwind css does not contain this class 'rotate-225'. Please change it with 'rotate-[225deg]'.

Reply:
Really Sorry About this.. 
Video: https://jam.dev/c/7f0608db-d97c-4987-bb05-c88b11c0f1ef